src/jdk.jfr/share/classes/jdk/jfr/internal/Repository.java

```diff
@@ -103,11 +104,20 @@ synchronized RepositoryChunk newChunk() {
         }
     }
 
+    private static LocalDateTime timestamp() {
+        try {
+            return LocalDateTime.now();
+        } catch (DateTimeException d) {
+            Logger.log(LogTag.JFR, LogLevel.INFO, "Could not create LocalDateTime with the default time zone. Using UTC time zone for chunk filename.");
+            return LocalDateTime.ofEpochSecond(System.currentTimeMillis(), 0, ZoneOffset.UTC);
```

Comment:
@egahlin 
**System.currentTimeMillis()** - returns count of milliseconds, but `LocalDateTime.ofEpochSecond` expects epoch **seconds** instead of millis.

Reply:
Good catch. I will file a bug to fix this.